en/views/json-and-xml-views.rst

```diff
@@ -162,7 +162,54 @@ Creating JSON Views
 The JsonView class supports the ``_jsonOptions`` variable that allows you to
 customize the bit-mask used to generate JSON. See the
 `json_encode <http://php.net/json_encode>`_ documentation for the valid
-values of this option.
+values of this option.::
+
+    // Controller code
+    class VideosController extends AppController
+    {
+    
+        //....
+        
+        public function export($acction = '')
+        {
+            $acction = mb_convert_case($acction, MB_CASE_LOWER);
+            
+            // Accion Out ..?
+            switch ($acction) {
+            
+                case 'json':
+                case 'xml':
+                    // Out file JSON ..?
+                    if($acction === 'json'){
```

Comment:
Code samples should follow the recommended coding standards. For example `if (...) {`